superset/views/core.py

```diff
@@ -891,6 +903,24 @@ def fetch_datasource_metadata(self) -> FlaskResponse:
         datasource.raise_for_access()
         return json_success(json.dumps(sanitize_datasource_data(datasource.data)))
 
+    @event_logger.log_this
+    @has_access
+    @expose("/language_pack/<lang>/")
```

Comment:
@mistercrunch It would be nice to put this endpoint under `/api/v1` to be an official endpoint.